package/utils/util-linux/Makefile

```diff
@@ -35,6 +35,31 @@ define Package/util-linux/Default
   URL:=http://www.kernel.org/pub/linux/utils/util-linux/
 endef
 
+define Package/util-linux-busybox
+$(call Package/util-linux/Default)
+  TITLE:=Full util-linux commands instead of busybox replacements
```

Comment:
Without the option it installs the packages from util-linux for which applets are normally built and since /usr/{bin,sbin} is earlier in the PATH the full versions will be present, although the busybox versions will still be present in {/bin,/sbin}